src/EventStream.js

```diff
@@ -23,23 +29,36 @@ class EventStream extends EventEmitter {
 			const req = requestor.request({
 				hostname,
 				protocol,
-				path: `${path}?history_limit=30&access_token=${this.token}`,
+				path: `${path}?access_token=${this.token}`,
```

Comment:
i realize the horse has left the barn already but we should really be sending this as a header

Reply:
It's a good question whether this works or not with the browser `stream-http` library in streaming mode. I'll give it a try.